iceoryx_binding_c/test/moduletests/test_client.cpp

```diff
@@ -488,5 +530,7 @@ TEST_F(iox_client_test, GetServiceDescriptionWorks)
     EXPECT_THAT(serviceDescription.serviceString, StrEq(SERVICE));
     EXPECT_THAT(serviceDescription.instanceString, StrEq(INSTANCE));
     EXPECT_THAT(serviceDescription.eventString, StrEq(EVENT));
+
+    iox_client_deinit(sut);
```

Comment:
Can we move this to the Teardown or Fixture Dtor or are there tests where no sut is created? 

Not necessary though, we need to get this done.

Reply:
Good idea but unfortunately I already merged this.